project/app/api/predict.py

```diff
@@ -94,6 +103,10 @@ async def load_models():
     GAS_MODELS['4'] = pickle.load(open(rockymnt, 'rb'))
     GAS_MODELS['5'] = pickle.load(open(westcoast, 'rb'))
 
+    global AIRBNB_MODEL
+    Airbnb_pred_model = os.path.join(os.getcwd(), 'app', 'airbnb_models', 'airbnb_model1.pkl')
+    AIRBNB_MODEL = pickle.load(open(Airbnb_pred_model, 'rb'))
```

Comment:
I think this might cause a problem right now, since we don't have your model in the repo, the pickle.load() won't have a file to read.

We might need to comment this out, or put some dummy code here.